python/pip_install/extract_wheels/lib/whl_filegroup_test.py

```diff
@@ -1,26 +1,52 @@
 import os
+import shutil
+import tempfile
+from typing import Optional
 import unittest
 
 from python.pip_install.extract_wheels.lib import bazel
 
 
-class TestExtractWheel(unittest.TestCase):
-    def test_generated_build_file_has_filegroup_target(self) -> None:
-        wheel_name = "example_minimal_package-0.0.1-py3-none-any.whl"
-        wheel_dir = "examples/wheel/"
-        wheel_path = wheel_dir + wheel_name
+class TestWhlFilegroup(unittest.TestCase):
+    def setUp(self) -> None:
+        self.wheel_name = "example_minimal_package-0.0.1-py3-none-any.whl"
+        self.wheel_dir = tempfile.mkdtemp()
```

Comment:
Moved the extraction into a temporary directory to keep the runfiles tree clean between test cases.